### PR TITLE
Ukrainian Translation - Core Templates

### PR DESCRIPTION
**Pull request by tymosh**
Makes progress on #1873

### CODE REVIEW

File: src/templates/uk/error.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Вказаний рік відсутній",
+    "Unsupported chapter requested": "Вказаний розділ відсутній",
+    "Unsupported language requested": "Вказана мова відсутня",
+    "Not Found": "Вказаний URL відсутній на сервері. Якщо ви вводили URL самі, перевірте правильність написання і спробуйте ще раз."
+  }
+%}
+{% macro expandedErrorMessage(errorCode, errorDescription, errorMessage) %}
+{{ errorCode }} {{ errorDescription }}: {{ errorMessage }}
```
Comment (bazzadp) [inline src/templates/uk/error.html:23]: The `errorDescription` comes from the server and is in English

```suggestion
{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
```

File: src/config/2019.json [hunk 2]
```diff
@@ -695,6 +695,14 @@
...
       "twitter": "KaJLa47"
     },
+    "tymosh": {
+      "name": "Kate Tymoshkina",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
+      "github": "tymosh"
+    },
```
Comment (bazzadp) [inline src/config/2019.json:705]: You happy with this @tymosh ? You can also add Website, Twitter and LinkedIn profiles if you want.

Here's [what it looks like on the site](https://20210116t081702-dot-webalmanac.uk.r.appspot.com/uk/2019/contributors#tymosh):

tymosh profile

File: src/config/2020.json [hunk 3]
```diff
@@ -757,6 +757,14 @@
...
       "twitter": "fox"
     },
+    "tymosh": {
+      "name": "Kate Tymoshkina",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
+      "github": "tymosh"
+    },
```
Comment (bazzadp) [inline src/config/2020.json:767]: Same here for 2020

File: src/server/language.py [hunk 4]
```diff
@@ -52,6 +52,7 @@ class Language(object):
     NL = _Language('Nederlands', 'nl')
     PT = _Language('Português', 'pt')
     RU = _Language('Русский', 'ru')
+    UK = _Language('Українська', 'uk')
```
Comment (bazzadp) [inline src/server/language.py:55]: Is this correct @tymosh ? I Googled it so apologies if wrong!

**Review by bazzadp — APPROVED** [review]
This is excellent @tymosh !!

I've moved it from `ua` to `uk` as it's the language code, not the country code.
I also enabled this in the config, and added your profile so you get credit on our contributors page.

I've put a test version here: https://20210116t081702-dot-webalmanac.uk.r.appspot.com/uk/2020/
Could you review this and make sure you're happy and then I think it's good to merge!

File: src/server/language.py (hunk 4 above)
Comment (tymosh) [reply src/server/language.py:55]: Yes, this is absolutely right

File: src/config/2019.json (hunk 2 above)
Comment (tymosh) [reply src/config/2019.json:705]: Please, add LinkedIn account: https://www.linkedin.com/in/tymosh/

File: src/config/2020.json (hunk 3 above)
Comment (tymosh) [reply src/config/2020.json:767]: Please, add LinkedIn account: https://www.linkedin.com/in/tymosh/

File: src/config/2019.json (hunk 2 above)
Comment (bazzadp) [reply src/config/2019.json:705]:
```suggestion
    "tymosh": {
      "name": "Kate Tymoshkina",
      "teams": [
        "translators"
      ],
      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
      "github": "tymosh",
      "linkedin": "tymosh"
    },
```

File: src/config/2020.json (hunk 3 above)
Comment (bazzadp) [reply src/config/2020.json:767]:
```suggestion
    "tymosh": {
      "name": "Kate Tymoshkina",
      "teams": [
        "translators"
      ],
      "avatar_url": "https://avatars2.githubusercontent.com/u/25080897?v=4&s=200",
      "github": "tymosh",
      "linkedin": "tymosh"
    },
```

**Comment by tymosh** [comment]
@bazzadp I've made some changes to the text. Everything else is ready to be merged.

**Comment by bazzadp** [comment]
Merged! Thanks so much for this.

We’ve got a few other changes almost ready to merge so will hold off doing a release until tomorrow evening after which you should see it live in the site!

**Pull request merged**